### ISSUE TITLE
dx11 pack stops beta5 from starting up

### ISSUE THREAD

**Issue by ggml** [mrvux/dx11-vvvv#295]
deleting report 294 for documentational reasons didnt make the issue go away, so here it is again.

added the 1.0.1 x64 pack to a clean install of beta5 and v4 wont start anymore.
(remove the pack and it works again)

regarding more details:
(please ask what you need so i dont have to reopen a third thread)

- specs on machine are win10 x64, 
- all setup is green
- antokhio says v4 does start after 5 minutes, i didnt wait for it. 

he produced this log:
> [13:43:50.445] everything you know is wrong
> [13:43:50.449] Adding lib\thirdparty\x64 to PATH.
> [13:43:51.095] creating SplashScreen
> [13:43:51.115] creating GClock
> [13:43:51.119] creating GMainloop
> [13:43:51.122] creating GLog
> [13:43:51.125] creating GApplication
> [13:43:51.128] creating GGlobals
> [13:43:51.131] creating device listener
> [13:43:51.134] creating HID device controller
> [13:43:51.324] creating GNodeFactory
> [13:43:51.328] creating Gex9Base
> [13:43:51.344] Direct3DEx initialized.
> [13:43:51.363] ExceptionDialog is disabled. Start vvvv with /showexceptions to enable it.
> [13:43:51.468] enumeratig Game Devices...
> [13:43:51.676] found Game Devices: "Controller (Xbox 360 Wireless Receiver for Windows)"
> [13:43:51.680] enumerating Midi Input Ports...
> [13:43:51.684] found Midi Input Ports: 
> [13:43:51.687] enumerating Midi Output Ports...
> [13:43:51.691] found Midi Output Ports: "Microsoft GS Wavetable Synth"
> [13:43:51.698] enumeratig TAPI Devices...
> [13:43:51.704] found TAPI devices: 
> [13:43:51.714] enumeratig Audio Renderer Devices...
> [13:43:51.757] found Audio Renderer Devices: "Speakers (High Definition Audio Device)","Decklink Audio Render","Default DirectSound Device","Default WaveOut Device","DirectSound: Optoma WXGA-8 (2- NVIDIA High Definition Audio)","DirectSound: Speakers (High Definition Audio Device)","DirectSound: HP LP2475w-C (2- NVIDIA High Definition Audio)","Optoma WXGA-8 (2- NVIDIA High Definition Audio)","HP LP2475w-C (2- NVIDIA High Definition Audio)"
> [13:43:51.761] enumeratig Midi Renderer Devices...
> [13:43:51.765] found Midi Renderer Devices: "Default MidiOut Device","Microsoft GS Wavetable Synth"
> [13:43:51.768] enumeratig External Renderer Devices...
> [13:43:51.772] found External Renderer Devices: "Decklink Video Render"
> [13:43:51.776] enumeratig Audio Input Devices...
> [13:43:51.780] found Audio Input Devices: "Decklink Audio Capture"
> [13:43:51.784] enumeratig Video Input Devices...
> [13:43:51.789] found Video Input Devices: "Decklink Video Capture","Dxtory Video 1","Dxtory Video 2","Dxtory Video 3","Dxtory Video 4"
> [13:43:51.813] creating MainForm
> [13:43:51.905] creating AddonHost
> [13:44:16.455] creating NodeBrowser, WindowSwitcher, Kommunikator
> [13:44:16.465] scanning for Externals
> [13:49:19.945] 0.v4p //// doesn't exist.
> [13:49:20.011] Starting up MainLoop
> [13:49:20.018] vvvv is up and running

**Comment by ggml** [comment]
joreg
not sure if it would solve the 5min delay but the idea is that a pack would contain a nodelist.xml so that vvvv does not have to parse the directory but can simply rely on the info in the nodelist.xml and typically this reduces startup-time.

@vux is there a reason you're not adding one for the dx11 pack? 
here is how: https://vvvv.org/documentation/nodelist-(vvvv)

**Comment by mrvux** [comment]
Yeah one liners will tend to get closed. Does not start is just way too vague and reasonably pointless, sorry...

So now there's a bit of details
* Does it do the same on 32 bits?:
* Does it do the same on another beta (33.7 or 34.2) ?
* Does it work if you delete VL from b35 install (basically delete the lib/packs folder, or just rename it to anything else)?
* Since you've seen the logstartup kindly provided by antokhio, you can run one as well (since it should show at which stage this gets off)?
* Does 0.7.1 pack has the same issue?

Thanks

**Comment by blausand** [comment]
Quick glance at the log suggests it's looking 5 minutes for a 0.v4p.
Possibly you have some junctions/hardlinks/symbolic links to big/remote drives or those of non-native filesystems that need ages to scan for 0.v4p?
Just a guess, becaue i'm tampering with _ext2fs_ and _hardlink shell extension_ a lot, because they're both damn useful and damn tricky.

**Comment by mrvux** [comment]
@blausand No, it's coming from the [13:44:16.465] scanning for Externals

But well considering the absence of information there's not much I can do there to be honest, only thing I know from several users is that b35 is generally slower to boot (plus using at least double the memory just from opening)

**Comment by ggml** [comment]
sorry for late, 

tried this:
- b32.4 x64 with 0.7.1 x64 (attached logstartup)
- b35.1 x64 with 1.0.1 x64
- b35.2 x86 with 1.0.1 x86 (attached logstartup) (tried rename lib/packs)
- b35.2 x64 with 1.0.1 x64 (attached logstartup) (tried rename lib/packs)

none worked with dx11 pack folder inside, all worked without it

all logstartups say "exception occurred while trying to create an instance of type 'VVVV.DX11.Factories.DX11NodesFactory'."

[screenshot](https://discourse.vvvv.org/uploads/default/optimized/2X/f/f1e14e781a78fb64a68ceafddc129a65091f7de5_1_800x644.PNG) of defender setup while still freezing

**Comment by ggml** [comment]
any idear what software might interfere in this case ?

**Comment by velcrome** [comment]
Hey @ggml saw your post over at the forum.

Just a stab in the dark, before you start wiping installs: what other packs do you have installed?

**Comment by ggml** [comment]
@velcrome only dx11
